redaxo/src/addons/structure/fragments/structure/scoped/category_row_header.php

```diff
@@ -0,0 +1,14 @@
+<?php
```

Comment:
hier ein beispiel scoped fragment.. dieses fragment ist genauso wie jedes andere auch.. einzige besonderheit, dass es in einem ordner 'scoped' beinhaltet ist, um zu signalisieren dass es nicht für wiederverwendung ausserhalb das packages (in diesem fall 'structure') gedacht ist